elements/package.json

```diff
@@ -33,6 +33,13 @@
     "./pf-clipboard-copy/pf-clipboard-copy.js": "./pf-clipboard-copy/pf-clipboard-copy.js",
     "./pf-code-block/BaseCodeBlock.js": "./pf-code-block/BaseCodeBlock.js",
     "./pf-code-block/pf-code-block.js": "./pf-code-block/pf-code-block.js",
+    "./pf-date-picker/date-picker-helper.js": "./pf-date-picker/date-picker-helper.js",
+    "./pf-date-picker/pf-calendar.js": "./pf-date-picker/pf-calendar.js",
+    "./pf-date-picker/pf-date-picker.js": "./pf-date-picker/pf-date-picker.js",
+    "./pf-date-picker/pf-month-select.js": "./pf-date-picker/pf-month-select.js",
+    "./pf-date-picker/pf-next-button.js": "./pf-date-picker/pf-next-button.js",
+    "./pf-date-picker/pf-previous-button.js": "./pf-date-picker/pf-previous-button.js",
+    "./pf-date-picker/pf-year-input.js": "./pf-date-picker/pf-year-input.js",
```

Comment:
since upstream only provides the `DatePicker` react component, we should also try to limit the number of public components we ship here. We may make an exception if, for accessibility reasons, we need to put these elements in the same root. However, in the case of date picker, We can probably allow ourselves the luxury of doing everything in the same shadow root. We'll need to investigate that carefully

Reply:
Done. 
Removed the child components exports list.